src/KeyboardInputSchemes.js

```diff
@@ -281,6 +282,10 @@ const VoiceOverInputScheme: KeyboardInputScheme = Object.assign({
     stopProgram: {
         keyDef: { code: "KeyS", key: "s", altKey: true},
         actionName: "stopProgram"
+    },
+    expandAddNode: {
```

Comment:
For now this is fine, but as we pick up more bindings (like the help key) that are the same between all schemes, we should probably make and reuse a "common" binding definition, like we do with extended keyboard sequences.  I think a TODO here would be a good idea for now.

Reply:
Current keyboard shortcut combinations are likely to change, so I think it makes sense to wait until we talk about rework of keyboard shortcut combinations.